programs/lib/syscalls/heap.c

```diff
@@ -0,0 +1,77 @@
+/**
+ * @author Magnus Mæhlum (magnmaeh@stud.ntnu.no)
+ * @brief A wrapper module to expose standard heap functions (malloc, free, etc.),
```

Comment:
I thought we were using newlib now? Here you use tinyalloc

Reply:
The file is not compiled, I just forgot to delete it